pdc/apps/release/routers.py

```diff
@@ -14,6 +14,8 @@
 router.register(r'releases', views.ReleaseViewSet)
 router.register(r'base-products', views.BaseProductViewSet)
 router.register(r'release-types', views.ReleaseTypeViewSet, base_name='releasetype')
+router.register(r'release-group-types', views.ReleaseGroupTypeViewSet, base_name='releasegrouptypes')
```

Comment:
Should PDC provide the api for release-group-types? Maybe I remember wrong. But In my impression, @sochotnicky mentioned no need to provide API for types.